.readthedocs.yml

```
@@ -9,5 +9,10 @@ python:
       path: .
       extra_requirements:
         - docs
+formats:
+  - epub
+  - pdf
+  - htmlzip
 sphinx:
+  configuration: docs/conf.py
```

Please simplify these changes, we don't need to specify the Sphinx configuration file, and we can just specify all formats.

```suggestion
formats: all
```